### DISCUSSION TITLE
Astrophysical application

### DISCUSSION THREAD

**Discussion by deSUBHAM** [BLAST-WarpX/warpx#3454] [Q&A] [answered]
Hi there. I am new to WarpX. I am interested in astrophysical problems. I am, therefore, wondering if there is any astrophysical application of WarpX. To be explicit, if it had been used for astrophysical problems. If so, would you please provide me some references?

**Comment by hklion** [comment, marked as answer]
What kinds of astrophysical problems are you interested in, and in particular, what physics would you want to include? We have ongoing efforts in astrophysics simulations, mainly related to magnetic reconnection and pulsar magnetospheres. The papers are currently in preparation, but we can share some slides from recent talks if that would be of interest.

**Reply by deSUBHAM** [reply]
Thank you very much for your reply. 

We are interested in topics related to magnetic reconnection, turbulence and particle acceleration. However, we would like to explore, particularly, the nonrelativistic regime. 

We will be happy if could share some of your slides as you mentioned.

**Reply by hklion** [reply]
Great, WarpX should be well-suited for that. We've mostly been working on relativistic reconnection, but it also works in non-relativistic regimes. If you send me an email at klion@lbl.gov I'll send you the slides. Please feel free to cc any of your colleagues who are also interested.
